content/en/code_analysis/troubleshooting/_index.md

```diff
@@ -60,10 +89,10 @@ It means that you are either:
 
 ### Results are not being surfaced in the Datadog UI
 
-If you are running Code Analysis on a non-GitHub repository, ensure that the first scan is ran on your default branch (for example, a branch name like
-`master` or `main`). After you commit on your default branch, non-default branches are analyzed.
+**If you are running Code Analysis on a non-GitHub repository**, ensure that the first scan is ran on your default branch (for example, a branch name like
```

Comment:
```suggestion
If you are using GitHub, ensure that you are using a GitHub App and have the required permissions. 

**If you are running Code Analysis on a non-GitHub repository**, ensure that the first scan is ran on your default branch (for example, a branch name like
```

Reply:
Usually the results don't show in the UI because they're not using GitHub, so I'd like to keep the original change